### DISCUSSION TITLE
Ubuntu (focal) still on 2.9.2

### DISCUSSION THREAD

**Discussion by krsche** [git-lfs/git-lfs#4684] [Q&A] [answered]
The latest available release for Ubuntu 20.04 LTS (Focal Fossa) is version `2.9.2-1` while 21.04 (Hirsute Hippo) is on `2.13.2`, see: https://launchpad.net/ubuntu/+source/git-lfs  

What is the update policy here?  
Unfortunately I could not find any info regarding that (also no mention in [docs/howto/release-git-lfs.md](https://github.com/git-lfs/git-lfs/blob/92ded33287780761aefafb7d0bbf060765c463b7/docs/howto/release-git-lfs.md).  
Adding the [packagecloud](https://packagecloud.io/github/git-lfs) repo would solve this problem of course, but I am wondering why ubuntu's default repo is still on such an old version?  

Many thanks in advance!

**Comment by bk2204** [comment, marked as answer]
Hey,

It's my understanding that most Linux distros, including Debian and Ubuntu, are not rolling release and therefore pick a single version and stick with it for the life of the distro.  This results in a more stable system at the expense of new features and bugfixes.  Security patches are typically applied as backports.

So there's essentially no way for us to update the versions in those distributions from what they had when they released.  In some cases, an older version may work just fine for you, and so you're free to use it if that's the case.  You could also use our packages or binaries, or build your own.

**Reply by krsche** [reply]
Thank you for the quick response! 
I wasn't aware of that. I guess I'll use your binaries then! 😃 

The matter that led me up this route was #3953 --> I got mislead by the output of `git lfs status` as well, luckily someone else improved it already; then noticed that the change was released as part of `2.10.0`, while I'm still on the version right before 😄 So I got curious why that is the case.  
Thanks for the clarification
